cmd/root_cmd.go

```diff
@@ -311,6 +317,8 @@ func ictestAddContract() *cobra.Command {
 
 	cmd.Flags().String(SuiteDirFlag, ".", "Path to the test suite directory. If not provided, the current working directory will be used.")
 	cmd.Flags().String(ContractNameFlag, "", "Name of the contract to be added to the test suite. If not provided, the contract name will be inferred from the schema file.")
+	cmd.Flags().MarkDeprecated(ContractNameFlag, "please use --package instead")
+	cmd.Flags().StringP(PackageFlag, "p", "", "Package name for the generated contract code. If not provided, the contract name will be inferred from the schema file.")
```

Comment:
We probably should use the pre-existing `PackageNameFlag`, so `--package-name` instead of `--package`. I forgot we had this flag when we had the discussions.

Reply:
@srdtrk 
Thanks for your suggestion! I deleted `PackageName` constant in the following commit.

[del: const PackageName](https://github.com/srdtrk/go-codegen/pull/90/commits/3dbc3b658472c3ab1de43841c048413879807691)